test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/AuthenticationTests.cs

```diff
@@ -294,14 +233,13 @@ public class AuthenticationTests
         [FrameworkSkipCondition(RuntimeFrameworks.CoreCLR, SkipReason = "HttpClientHandler issue (https://github.com/dotnet/corefx/issues/5045).")]
         public async Task AuthTypes_ChallengeWithoutAuthTypes_AllChallengesSent(AuthenticationSchemes authType)
         {
-            string address;
             var authTypeList = authType.ToString().Split(new char[] { ',', ' ' }, StringSplitOptions.RemoveEmptyEntries);
-            using (Utilities.CreateHttpAuthServer(authType, AllowAnoymous, out address, httpContext =>
+            using (var server = Utilities.CreateDynamicHost(string.Empty, authType, AllowAnoymous, out var address, out var baseAddress, httpContext =>
             {
                 Assert.NotNull(httpContext.User);
                 Assert.NotNull(httpContext.User.Identity);
                 Assert.False(httpContext.User.Identity.IsAuthenticated);
-                return httpContext.Authentication.ChallengeAsync();
+                return httpContext.ChallengeAsync(authType.ToString());
```

Comment:
Yeah what's the usage scenario, would that work?  Can it be considered a "Host" scheme, and translate that into the AuthenticationSchemes inside of its options?  Or do you expect ppl to actually put Authorize(NTLM/Basic/Negotitate)?